biocypher_metta/adapters/uberon_adapter.py

```diff
@@ -0,0 +1,75 @@
+# http://purl.obolibrary.org/obo/UBERON_0009621,tail somite,,A somite that is part of a tail.,false,,,http://purl.obolibrary.org/obo/UBERON_0002329,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,XAO:0003143|EMAPA:16860|AAO:0010383,,,A somite that is part of a tail.,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,uberon,,,,,,,,,,,,http://purl.obolibrary.org/obo/UBERON_0002415,,http://purl.obolibrary.org/obo/UBERON_0002415,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,tail somite,,,UBERON:0009621,,,,,UBERON:0009621,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+# http://purl.obolibrary.org/obo/UBERON_0009624,lumbar nerve,nervi lumbales|nervus lumbalis|lumbar spinal nerve,The lumbar nerves are the five spinal nerves emerging from the lumbar vertebrae. They are divided into posterior and anterior divisions.,false,,,http://purl.obolibrary.org/obo/UBERON_0001780,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,neuronames:1730|Wikipedia:Lumbar_nerves|UMLS:C0228897|SCTID:361600002|NCIT:C33015|FMA:5861,,,The lumbar nerves are the five spinal nerves emerging from the lumbar vertebrae. They are divided into posterior and anterior divisions.,,,http://upload.wikimedia.org/wikipedia/commons/d/d1/Gray802.png,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,http://purl.obolibrary.org/obo/UBERON_0002792,,,,,,,,,,,,,,,,,,,,,,,,,,,nervus lumbalis|lumbar spinal nerve,,,,,,,,,,,,,,,,,,,,,,,,,,,nervi lumbales,,,,,,,,,,,,,,uberon,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,lumbar nerve,,,UBERON:0009624,,,,,UBERON:0009624,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+
+class UberonAdapter(OntologyAdapter):
```

Comment:
Why are you inheriting from `OntologyAdapter` if you aren't using any of its methods?